### ISSUE TITLE
Remake as cards

### ISSUE THREAD

**Issue by shikha-16** [shikha-16/Women-in-Technology#139]
Remake all the entries in 'Articles & Blogs' and 'Github Repositories to Star' and 'Infographics' as cards in 3 different sections of the Other Resources page. (And delete the existing things)
![Screenshot from 2021-09-03 00-12-39](https://user-images.githubusercontent.com/68149849/131899423-dd0e85d7-128f-4f8d-8f8f-d4730d2833ab.png)
![Screenshot from 2021-09-03 00-15-14](https://user-images.githubusercontent.com/68149849/131899458-d312ce1f-06d2-4657-96c4-b276de22687e.png)
![Screenshot from 2021-09-03 00-15-33](https://user-images.githubusercontent.com/68149849/131899488-0b80da74-70ed-46f3-a758-38dd64049453.png)

**Comment by nayabatir1** [comment]
may I help

**Comment by shikha-16** [comment]
> may I help

Yes sure, please connect with me on discord as I would like to talk to you about this in more detail.

**Comment by madhhuurrii** [comment]
Hi @shikha-16, I would like to work on this issue if it is available!

**Comment by shikha-16** [comment]
> Hi @shikha-16, I would like to work on this issue if it is available!

Okay, I am assigning you to this @madhhuurrii 
@nayabatir1 the other issue is already assigned to you and waiting so giving this one to Madhuri.

**Comment by madhhuurrii** [comment]
@shikha-16 I wanted some clarification regarding the idea, should the "Study Resource" Page contain all the 4 section as it is or else should design and place them in a card?

**Comment by shikha-16** [comment]
> @shikha-16 I wanted some clarification regarding the idea, should the "Study Resource" Page contain all the 4 section as it is or else should design and place them in a card?

so, make 4 different sections on the Study Resources page. The contents of each section should be placed in cards. And please make the cards attractive to look at as well!

**Comment by shikha-16** [comment]
> Hii @shikha-16 , So I did change the layout of the study resource a bit and remade it as cards. You can have look, Is this fine or do you want any other card design? ![Study Resource page](https://user-images.githubusercontent.com/64744084/136453879-d000ac81-6850-404c-87a0-7123d8708fbb.png)
> 
> If you want I can make the Infographic card similar to the above two section

Hey, this looks good! So I want all 4 sections to have the same layout. I like the cards in 'Articles' and 'github repos', so I would want that type for all 4 sections. Just one change though, can you make the cards such that there are 2 cards in a row, for all the 4 sections? And make sure the cards are properly spaced and center aligned.
Also put the heading 'Infographics' in the last section.

Thanks!!

**Comment by madhhuurrii** [comment]
> > Hii @shikha-16 , So I did change the layout of the study resource a bit and remade it as cards. You can have look, Is this fine or do you want any other card design? ![Study Resource page](https://user-images.githubusercontent.com/64744084/136453879-d000ac81-6850-404c-87a0-7123d8708fbb.png)
> > If you want I can make the Infographic card similar to the above two section
> 
> Hey, this looks good! So I want all 4 sections to have the same layout. I like the cards in 'Articles' and 'github repos', so I would want that type for all 4 sections. Just one change though, can you make the cards such that there are 2 cards in a row, for all the 4 sections? And make sure the cards are properly spaced and center aligned. Also put the heading 'Infographics' in the last section.
> 
> Thanks!!

Okay! will do this changes

**Comment by madhhuurrii** [comment]
Hii @shikha-16 ,  Have a look at this layout
I felt like the Explore CS section is good now if I try to make that as a card, It might look more monotonous page.
![studyresource](https://user-images.githubusercontent.com/64744084/136619184-94c26b32-ef55-4828-840b-4f4f502eb9f7.png)
